tests/cli/test_rerun.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
```

Comment:
why is this part of this PR?

Reply:
Cause we need to refactor it anyway and since I entered regression with this PR I decided to left it here. If it bothers you I can make a its own PR for this.

Reply:
Looks like my comment got deleted. :(